torch/csrc/api/include/torch/mps.h

```diff
@@ -17,5 +27,15 @@ void TORCH_API manual_seed(uint64_t seed);
 /// Waits for all streams on a MPS device to complete.
 void TORCH_API synchronize();
 
+/// Submits the command buffer to run on the GPU
+void TORCH_API commit();
+
+/// Get the current command buffer to encode the Metal commands
+MTLCommandBuffer_t TORCH_API get_command_buffer();
```

Comment:
@razarmehr , this is something we wanted to avoid. As these are typedef based hacks.

Reply:
Either the code is C/C++, or Objective-C, and you have to check for it with that flag. There's no third option.
We're doing it so users don't have to cast in their side. It looks clean and transparent to users:
`id<MTLCommandBuffer> commandBuffer = torch::mps::get_command_buffer();`

Please let me know your suggestion, and I'll make change.

Reply:
Yeah, Nikita is fine with it, so I am not concerned.
I am aware of this typedef hack as I had added it long back to MPSStream.h, I just was hoping we can do a better option. The suggestion I have in mind, is to including the files in ObjC path only. This will avoid the build failure.